var/spack/repos/builtin/packages/r-dexseq/package.py

```diff
@@ -17,15 +17,18 @@ class RDexseq(RPackage):
     and generalized linear models for testing. The package also provides
     functions for the visualization and exploration of the results."""
 
-    homepage = "http://www.bioconductor.org/packages/release/bioc/html/DEXSeq.html"
-    url      = "http://www.bioconductor.org/packages/release/bioc/src/contrib/DEXSeq_1.36.0.tar.gz"
-    bioc     = "DEXSeq"
+    git  = "https://github.com/areyesq89/DEXSeq.git"
+    bioc = "DEXSeq"
 
     maintainers = ['dorton21']
 
-    version('1.36.0', sha256='111b107cd70f770d1c6f1d5d6d2e447eb61008398b14a390979c64d60559d271')
+    version('1.36.0', commit='ecc9bca9961ad2c2414703cdf8fc8bcb38c9b729')
```

Comment:
This is the wrong commit, because the repo it comes from is older.
```suggestion
    version('1.36.0', commit='f0a361af6954fcc2abb2db801c26e303570669b2')
```
I also realized that if you already had the tarfile downloaded the git information is in the DESCRIPTION file.
```
git_url: https://git.bioconductor.org/packages/DEXSeq
git_branch: RELEASE_3_12
git_last_commit: f0a361a
git_last_commit_date: 2020-10-27
```